_includes/pbc/all/install-features/202307.0/install-the-dynamic-store-product-feature.md

```diff
@@ -0,0 +1,62 @@
+{% info_block warningBox %}
+
+Dynamic Multistore is currently running under an Early Access Release. Early Access Releases are subject to specific legal terms, they are unsupported and do not provide production-ready SLAs. They can also be deprecated without a General Availability Release. Nevertheless, we welcome feedback from early adopters on these cutting-edge, exploratory features.
+
+{% endinfo_block %} 
+
+This document describes how to integrate the  Dynamic Store + Product feature into a Spryker project.
+
+## Install feature core
+
+### Prerequisites
+
+To start feature integration, overview and install the necessary features:
+
+| NAME | VERSION |
+| --- | --- |
+| Spryker Core | {{page.version}} |
+| Product | {{page.version}} |
+
+### Set up behavior
+
+Register the following plugins:
+
+| PLUGIN | SPECIFICATION | PREREQUISITES | NAMESPACE |
+| --- | --- | --- | --- |
+| StoreProductCategoryListActionViewDataExpanderPlugin | Expands view data for list of product categories with stores data. | None | Spryker\Zed\StoreGui\Communication\Plugin\ProductCategoryFilterGui |
```

Comment:
Any particular place where is list of product categories is displayed?

Reply:
it should be at the /product-category-filter-gui page at the backoffice as I can see